pyproject.toml

```diff
@@ -1,20 +1,8 @@
 [tool.black]
 line-length = 100
-# TODO: Use `--extend-exclude` once we update black version to >= 21.4b0
-exclude = '''
+target-version = ['py36']
+force-exclude = '''
 /(
-  # default exclude
```

Comment:
this new version of black has a config that ignores all of these automatically?

Reply:
Yes, in older versions of `black`, `exclude` option overwrites the default `exclude`.

Reply:
If we specify `exclude`, it overwrites the default value of `exclude`.